lib/data/data_sources/local_data_source.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:hive_flutter/hive_flutter.dart';
+import 'package:marvel/data/models/local_models/local_character.dart';
+
+abstract class LocalDataSource {
+  Future<void> charactersToDB(LocalCharacter character);
```

Comment:
```suggestion
  Future<void> save(LocalCharacter character);
```

Reply:
Name should be short and at the same time clearly enclose its intention.

Reply:
done